### ISSUE TITLE
no-unused-variable doesn't properly detect usage of lowercase imports in TSX files

### ISSUE THREAD

**Issue by prencher** [palantir/tslint#1716]
### Bug Report

- __TSLint version__: `3.15.1`
- __TypeScript version__: `1.8.7`
- __Running TSLint via__: Visual Studio Code 1.7.1

#### TypeScript code being linted:

```tsx
import * as menuItems from './menuItems';

class Foo {
    render() {
        return <menuItems.Delete />;
    }
}
```

#### `tslint.json` configuration:
```json
{
    "rules": {
        "no-unused-variable": [
            true,
            "react"
        ]
   }
}
```

#### Actual behavior

Shows `menuItems` as unused.

#### Expected behavior

Not to do that.

**Comment by adidahiya** [comment]
duplicate of https://github.com/palantir/tslint/issues/1010. the fix is to upgrade your typescript version.
